packages/trader/src/Modules/Trading/Components/Form/RadioGroupWithInfoMobile/radio-group-with-info-mobile.tsx

```diff
@@ -47,18 +49,20 @@ const RadioGroupWithInfoMobile = ({
 
     return (
         <>
-            <div className={`trade-params__${contract_name}-ic-info-wrapper`}>
-                <Popover
-                    alignment={popover_alignment}
-                    classNameBubble='dc-popover__trade-params'
-                    disable_target_icon={is_tooltip_disabled}
-                    icon='info'
-                    id={`dt_${contract_name}-stake__tooltip`}
-                    is_bubble_hover_enabled
-                    zIndex='9999'
-                    message={info}
-                />
-            </div>
+            {should_show_tooltip && (
```

Comment:
Made render of this popover optional. In order not to break existing usage I make it `true` by default.